pkg/transformer/openshift/openshift.go

```diff
@@ -202,11 +203,20 @@ func (o *OpenShift) initImageStream(name string, service kobject.ServiceConfig,
 func initBuildConfig(name string, service kobject.ServiceConfig, repo string, branch string) (*buildapi.BuildConfig, error) {
 	contextDir, err := getAbsBuildContext(service.Build)
 	envList := []kapi.EnvVar{}
-	for envName, envValue := range service.BuildArgs {
-		if *envValue == "\x00" {
-			*envValue = os.Getenv(envName)
+	var sortEnvName []string
```

Comment:
Why are we changing how we generate initBuildConfig in a PR for testing functional tests?

This should be changed in a separate PR.